docs/_series/phockpark.md

```diff
@@ -3,7 +3,7 @@ name: PHOCKPARK
 author: Pepe Peddler
 image: https://arweave.net/dtBClMX7YbP4M7QYvZ0GldFRjssimxUWMisKtzgapSM/aw63r3_image.gif
 date: 2022-02-23
-description: A PHOCKHEAD Series Parody of 50 your favorite Characters from SP. Currently in distribution Phase
+description: A PHOCKHEAD Series Parody of 50 your favorite Characters from SP. Currently in distribution Phase : Grail holders will recieve all 50 characters as a dividends once all grail cards are distributed. 
```

Comment:
If you use any special characters (such as colon ":" ) the whole text needs to be in single or double quotes.